ui/webui/src/components/time/DateAndTime.jsx

```diff
@@ -0,0 +1,505 @@
+                                <Flex
+                                  direction={{ default: "row" }}
+                                  spaceItems={{ default: "spaceItemsSm" }}
+                                >
+                                    <FormGroup
+                                      label={_("Date")}
+                                      isRequired
+                                      fieldId="time-date-edit-date"
+                                    >
+                                        <DatePicker
+                                          isDisabled={NTPSyncEnabled}
+                                          dateFormat={timeformat.date}
+                                          dateParse={timeformat.parseDate}
+                                          value={timeformat.date(systemDate.valueOf())}
+                                          onChange={handleDateChange}
+                                        />
+                                    </FormGroup>
+                                    <FormGroup
+                                      label={_("Time")}
+                                      isRequired
+                                      fieldId="time-date-edit-time"
+                                    >
+                                        <TimePicker
+                                          isDisabled={NTPSyncEnabled}
+                                          time={systemDate}
+                                          onChange={handleTimeChange}
+                                        />
+                                    </FormGroup>
+                                </Flex>
+                            </GridItem>
+                        </Grid>
+                    </FormSection>
+                    <Divider />
+                    <Text className="time-edit-automatic-hint">
+                        {_("To edit date and time, turn off automatic time")}
+                    </Text>
+                    <ActionGroup>
+                        <Button
+                          id={idPrefix + "-btn-save"}
+                          variant="primary"
+                          onClick={handleSave}
+                        >
+                            {_("Save")}
+                        </Button>
+                        <Button
+                          id={idPrefix + "-btn-cancel"}
+                          variant="link"
+                          onClick={handleCancel}
+                        >
+                            Cancel
+                        </Button>
+                    </ActionGroup>
+                </Form>
+            </CardBody>
+        </Card>
+    );
+};
+
+const AutomaticTimeNotification = ({ automaticTimeEnabled }) => {
+    return (
+        <Flex direction={{ default: "row" }} spaceItems={{ default: "spaceItemsSm" }} alignSelf={{ default: "alignItemsCenter" }}>
+            <FlexItem>
+                <Icon status="info">
```

Comment:
The mockup is using an info alert.

<Alert variant="info" isInline isPlain title="Info inline alert title" />

https://www.patternfly.org/v4/components/alert#plain-inline-alert-variants